src/pages/DetailBlogpost.jsx

```diff
@@ -0,0 +1,29 @@
+import {Link, useParams} from "react-router-dom";
+import posts from "../constants/data.json"
+import {dateFormat} from "../helpers/dateFormat.js";
+
+const DetailBlogpost = () => {
+    const { id} = useParams();
+
+    const newId = posts.find(post => {
+        return post.id.toString() === id;
+    })
+    console.log(newId)
+
+
+    return (
+        <>
```

Comment:
Heb je `<></>` hier wel nodig? De `<article>`-tag is volgens mij voldoende als omwikkelend element?